hack/jenkins/e2e.sh

```diff
@@ -49,29 +53,48 @@ echo "E2E_OPT: ${E2E_OPT}"                         # hack/e2e.go options
 echo "E2E_SET_CLUSTER_API_VERSION: ${E2E_SET_CLUSTER_API_VERSION:-<not set>}" # optional, for GKE, set CLUSTER_API_VERSION to git hash
 echo "--------------------------------------------------------------------------------"
 
-# GCE variables
-export INSTANCE_PREFIX=${E2E_CLUSTER_NAME}
-export KUBE_GCE_ZONE=${E2E_ZONE}
-export KUBE_GCE_NETWORK=${E2E_NETWORK}
 
-# GKE variables
-export CLUSTER_NAME=${E2E_CLUSTER_NAME}
-export ZONE=${E2E_ZONE}
-export KUBE_GKE_NETWORK=${E2E_NETWORK}
+if [[ "${KUBERNETES_PROVIDER}" == "aws" ]]; then
```

Comment:
Is there any harm in setting all the variables? I've actually been taking the style of setting all variables for all providers (you see GCE and GKE set both).